community/cypher/cypher/src/test/scala/org/neo4j/cypher/NotificationAcceptanceTest.scala

```diff
@@ -38,7 +38,7 @@ class NotificationAcceptanceTest extends ExecutionEngineFunSuite with NewPlanner
   }
 
   test("Warn for cartesian product with runtime=interpreted") {
-    val result = executeWithAllPlannersAndCompatibilityMode("explain cypher runtime=interpreted match (a)-->(b), (c)-->(d) return *")
+    val result = innerExecute("explain cypher runtime=interpreted match (a)-->(b), (c)-->(d) return *")
```

Comment:
Because I have have changed so that all queries running `executeWithAllPlannersAndCompatibilityMode` first runs the query with `runtime=compile` and running the query with both `interpreted` and `compiled` causes confusion.

The other changes in this file is because we will get  `RuntimeOperationNotSupported` notification now that we run with `compiled=runtime` for queries that are not supported which changes all the assertions and make the tests more confusing.

For these queries we could probably change it to just use the normal execute since all queries run with `EXPLAIN` meaning that `executeWithAllPlannersAndCompatibilityMode` doesn't get any results to compare and will only return the notifications from the default mode and does no extra checks to verify that all modes return the same notifications.